api/rds.proto

```diff
@@ -472,7 +480,10 @@ message RateLimit {
   // Refers to the stage set in the filter. The rate limit configuration only
   // applies to filters with the same stage number. The default stage number is
   // 0.
-  // NOTE: The filter supports a range of 0 - 10 inclusively for stage numbers.
+  //
+  // .. note::
+  //
+  //   The filter supports a range of 0 - 10 inclusively for stage numbers.
   google.protobuf.UInt32Value stage = 1;
```

Comment:
Could add constraints here.

Reply:
I didn't do rate limit yet. I will do this in a follow up pass.